.templates/php/default.mk

```diff
@@ -33,20 +48,21 @@ post-release:
 .PHONY: post-release
 
 clean:
+	rm -rf .tested .deps .codegen
 	rm -rf vendor composer.lock
 .PHONY: clean
 
-.tested: .deps .codegen $(PHP_SOURCE_FILES)
+.tested: .deps $(PHP_SOURCE_FILES)
+	vendor/bin/php-cs-fixer --dry-run --diff fix
+	vendor/bin/psalm --no-cache
 	vendor/bin/phpunit
-	vendor/bin/psalm
-.PHONY: .tested
+	touch $@
 
-.deps: composer.lock
+.deps: vendor .codegen
 	touch $@
 
 .codegen:
 	touch $@
 
-composer.lock: composer.json
+vendor: composer.json
 	composer install
-	touch $@
```

Comment:
Because the make target is `vendor` and it's created by the commands that are run

Reply:
It's not needed as the commands create the `vendor` folder

Reply:
yes but sometimes the command does not correctly refreshes the timestamp of the directory hence the necessity of the touch since makefiles relies on the timestamp